src/main/resources/hudson/plugins/active_directory/ActiveDirectorySecurityRealm/configAdvanced.jelly

```diff
@@ -1,6 +1,9 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
   <f:advanced>
+    <f:entry field="userFromHTTPHeader" title="${%Use HTTP-Header as username provider. Example: 'X-Forwarded-User'}">
```

Comment:
This would be a gnarly property name to type in a translation. You can make a property name for the L10n string in `configAdvanced.properties` to make that string the default English translation. Feel free to move the other descriptions in this file there if you'd like, though simple L10n strings aren't that difficult to encode as properties file keys.